packages/xdefi/src/lib/xdefi.ts

```diff
@@ -146,6 +146,10 @@ const XDEFI: WalletBehaviourFactory<InjectedWallet> = async ({
 
       return result;
     },
+
+    async importAccountsInSecureContext(params) {
+      _state.wallet.importAccounts(params.accounts);
```

Comment:
if it passes just accounts info without `network` how would it aware if account is from `testnet` or `mainnet`?

Reply:
Wallet itself will take current network and use it for migration.